src/content/documentation/develop/firefox-builtin-data-consent.md

```diff
@@ -0,0 +1,248 @@
+
+![Figure 2: The data permissions are also listed in about:addons](/assets/img/documentation/develop/data-collection-permissions-about-addons.webp)
+*Figure 2: The data permissions are also listed in about:addons*
+
+#### Optional data
+
+Optional data collection permissions can be specified using the optional list. These are not surfaced during installation (except `technicalAndInteraction`; see next section), and they are not granted by default. The extension can request the user opts in to this data collection after installation via a prompt, and the user can enable or disable this option data collection at any time in `about:addons` in the *Permissions and data* section of the extension settings.
+
+### Technical and interaction data
+
+The `technicalAndInteraction` data type behaves differently compared to all others. This data permission can only be optional, but unlike other optional data collection options the user has the opportunity to enable or disable this during the installation flow. In Figure 1 above, this choice is available in the optional settings section of the installation prompt.
+
+### No data collection
+
+If an extension does not collect or transmit any data, developers should explicitly indicate that by specifying the `none` required permission in the manifest, as follows:
+
+```json
+{
+  "manifest_version": 2,
+  "name": "extension without data collection",
+  "version": "1.0.0",
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "@extension-without-data-collection",
+      "data_collection_permissions": {
+        "required": ["none"]
+      }
+    }
+  },
+  "permissions": [
+    "bookmarks",
+    "<all_urls>"
+  ]
+}
+```
+
+When a user attempts to install this extension, Firefox will show the usual installation prompt with the description of the required (API) permissions as well as a new description to indicate that the extension does not collect any data (see Figure 3).
+
+![Figure 3: Installation prompt with no data transmission defined in the manifest](/assets/img/documentation/develop/data-collection-permissions-prompt-install-no-transmission.webp)
+*Figure 3: Installation prompt with no data transmission defined in the manifest*
+
+The "no data collected" type is also listed in the *Permissions and data* tab of the extension in `about:addons` as shown in Figure 4.
+
+![Figure 4: The "no data collected" permission is listed in about:addons](/assets/img/documentation/develop/data-collection-permissions-about-addons-no-transmission.webp)
+*Figure 4: The "no data collected" permission is listed in about:addons*
+
+Note: The `none` data type can only be required, and it cannot be used with other data types, including optional types. When that happens, Firefox will ignore the `none` type, and only consider the other data types (see next section for more information). In addition, Firefox will show a warning message intended to developers in `about:debugging` as shown in Figure 5.
```

Comment:
What does that mean? Optional-only data collection is possible.

Reply:
I realize this is coming from the blog post. Firefox currently allows no required data collection + optional data collection perms, which would result in the following for example:

```
{ "required": ["none"], "optional": ["healthInfo"] }
```

Reply:
Will and I chatted about this directly and agreed to remove this part for now, determine what the expected behavior should be and document that, after a potential implementation update, if necessary.